package.json

```diff
@@ -21,25 +21,32 @@
     "clean:contracts": "rimraf ./build/contracts/*",
     "clean:ganache": "rimraf ./ganache-chain-db/",
     "provision:token:contracts": "truffle compile && truffle compile --contracts_directory 'lib/dappsys/[!note][!stop][!proxy][!thing][!token]*.sol' && bash ./scripts/provision-token-contracts.sh",
+    "provision:safe:contracts": "truffle compile && truffle compile --contracts_directory 'lib/safe-contracts/contracts/proxies/*.sol' && truffle compile --contracts_directory 'lib/safe-contracts/contracts/*.sol' && bash ./scripts/provision-safe-contracts.sh",
+    "start:bridging": "cd ./lib/safe-contracts && npm i || true && cd ../../ && bash ./scripts/start-bridging-environment.sh && node ./scripts/start-bridging-environment.js &",
     "start:blockchain:client": "bash ./scripts/start-blockchain-client.sh",
+    "start:blockchain:client:2": "CHAIN_ID=2656692 PORT=8546 DBPATH=./ganache-chain-db-2 bash ./scripts/start-blockchain-client.sh",
```

Comment:
Correct. `ganache` uses a chain id of 1337, otherwise. I'll note as well that we have a [default](https://github.com/JoinColony/colonyNetwork/blob/develop/scripts/start-blockchain-client.sh#L6) for the chainId, so we've been hard coding it all along, just not here.